Config.psm1

```diff
@@ -31,7 +31,7 @@ function Get-AvailableConfigOptions {
         @{"Name" = "virtual_disk_format"; "DefaultValue" = "VHDX";
```

Comment:
Change commit message to:

Added VMware image_type

Here write the constraints needed to create a vmware image and what is the output format.